index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav class="navbar" id="nav-bar">     
+      <img src="images/product-landing-page-logo.png" alt="Header Image" id="header-img">
+      <ul class="nav-link">
+        <li><a href="#features">Features</a></li>
+        <li><a href="#how-it-works">How it works</a></li>
+        <li><a href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>
+    <section class="heading" id="header">
+      <h1>Handcrafted, home-made masterpieces</h1>
+    </section>
+    <form action="https://www.freecodecamp.com/email-submit" id="form">
+      <input type="email" placeholder="email">
+      <input type="submit" id="submit">
+    </form>
+    <div class="container">
+    <section id="features">
+      <h1>Features</h1>
```

Comment:
Having multiple `<h1>` tags is an error

Reply:
> Having multiple `<h1>` tags is an error

Please suggest your requested changes politely and includes an explanation for them. For example, in this case, you can explain why having multiple `<h1>` tags is an error. In this way, you will communicate more effectively and it will be easier to understand and accomplish the changes. You can follow [these steps](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) to understand more about code reviews.